assets/js/pages/Users/UsersPage.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { listUsers, deleteUser } from '@lib/api/users';
+
+import { toast } from 'react-hot-toast';
+
+import Users from './Users';
+
+const successDeleteMessage = 'User deleted successfully';
```

Comment:
I would put this constants as upper case `SUCCESS_DELETE_MESSAGE` and so on.
At the end they are real consts

Reply:
+1